deliveries/bluesky_milestone_1.md

```diff
@@ -0,0 +1,44 @@
+# Milestone Delivery :mailbox:
+
+> ⚡ Only the GitHub account that submitted the application is allowed to submit milestones. 
+> 
+> Don't remove any of the mandatory parts presented in bold letters or as headlines! Lines starting with `>`, such as this one, can be removed.
+
+**The invoice has been send out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/smart-contract-bounty/Support-Docs/blob/master/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:**, please provide a link to the merged contract from the running buonty (the `.md` file in the [applications](https://github.com/smart-contract-bounty/Wasm-Bounty-01/tree/master/applications) directory).
+
+[Bluesky project]()
+
+**Context** (optional)
+> Please provide a short paragraph or two connecting the deliverables in this milestone and describing their purpose.
+
+In this [technical showcase](https://medium.com/@opensmartcontract/learn-ink-by-example-order-food-on-blockchain-a4024b2dee4a) we implemented with ink! smart contract language including custom macros, security, testing scripts, deployment and documents. It delivers the first milestone: define a specific use case: Order Food on Blockchain, implement in ink! with macros encapsulating reusable business logic, tested and deployed, then publish article to share with the community learning experiences / resources / recommendations about ink! and openBrush. 
+
+**Deliverables**
+> Please provide a list of all deliverables of the milestone extracted from the initial application and a link to the deliverable itself. Ideally all links inside the below table should include a commit hash, which will be used for testing. If you don't provide a commit hash, we will work off the default branch of your repository. Thus, if you plan on continuing work after delivery, we suggest you create a separate branch for either the delivery or your continuing work. 
+> 
+> If there is anything particular about any of the deliverables we or a future reader should know, use the respective `Notes` column.
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 1. | Define use case | [Food Order on Blockchain](https://medium.com/@opensmartcontract/learn-ink-by-example-order-food-on-blockchain-a4024b2dee4a) | define a contrived use case with [persona](https://docs.google.com/document/d/1yiZrPnSaz9myrXOrR2k2IALbJA9XcjeEJ-wCmhn3S3o/edit#heading=h.s7qu46re6cic), [workflow](https://docs.google.com/document/d/1yiZrPnSaz9myrXOrR2k2IALbJA9XcjeEJ-wCmhn3S3o/edit#heading=h.p52xmq7d7g3m), [target audiences](https://docs.google.com/document/d/1yiZrPnSaz9myrXOrR2k2IALbJA9XcjeEJ-wCmhn3S3o/edit#heading=h.y8aexxoud7oj) for ink! smart contract implementation |
```

Comment:
Here is some of my feedback:

* The Medium article you posted has tons of grammatical errors. Also, `Kevin Woods` should be `Dr. Gavin Woods`. 

* Consider using ChatGPT to fix grammatical errors

* Suggestion: Consider changing the title of the article. Maybe something like "Build a Food Ordering dApp with ink!" or "Learn ink! by Example — Order Food on the Blockchain" (you need a "the" before Blockchain)

Reply:
Sorry for those errors. Great advice. Fixed the title, name, grammatical errors, then updated both the [Gdoc](https://docs.google.com/document/d/1yiZrPnSaz9myrXOrR2k2IALbJA9XcjeEJ-wCmhn3S3o/edit#heading=h.qqu7wcxgy9pd), and the [Medium article](https://medium.com/@opensmartcontract/learn-ink-by-example-order-food-on-blockchain-a4024b2dee4a) accordingly.